README.md

```diff
@@ -425,6 +429,24 @@ Furthermore, CTL exposes an `overlay` from its flake. You can use this in the Ni
 
 We have recenly set up a small scaffolding repository for projects wishing to adopt CTL: https://github.com/mlabs-haskell/ctl-scaffold. More documentation and resources will be added soon to the repo
 
+### Changing network configurations.
+
+CTL supports using networks other than testnet to provide development environment.
+
+First you need to specify an alternative way of getting the network configuration.
+
+[cardano-configurations](https://github.com/input-output-hk/cardano-configurations) repo provides configs for `mainnet`, `staging` and a few other networks.
+
+Which of the network config directories is chosen is determined by `network.name` parameter of `buildCtlRuntime`
+
+You can specify your own fork of `cardano-configurations` like this:
+
+```nix
+inputs.cardano-transaction-lib.inputs.cardano-configurations.follows = "...";
+```
+
+When changing networks, make sure that `network.magic` is correctly synchronized with value in config (see `protocolConsts.protocolMagic` in `byron.json`).
```

Comment:
No, because we need to distinguish between mainnet and other networks (see comment by @ngua about ctl-server `--network-id`), and without full knowledge about all protocol parameters we can't determine that - network magic is just a nonce by itself.